### PR TITLE
Add and check license headers

### PR DESCRIPTION
**Pull request by turbolent**
All source files should have a license header. Add it to all files and add a CI step to ensure we don' forget to add them.

### CODE REVIEW

**Comment by MaxStalker** [comment]
Shall we also upgrade license type here? 🤔
https://github.com/onflow/flow-js-testing/blob/7afda575edeacc858f8b61525360a17bd7be45dc/package.json#L17

**Comment by turbolent** [comment]
@MaxStalker Yes, good catch!

File: check-headers.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+files=$(find src -name \*.js -type f -print0 | xargs -0 egrep -L '(Licensed under the Apache License)|(Code generated from|by)')
+if [ -n "$files" ]; then
+  echo "Missing license header in:"
+  echo "$files"
+  exit 1
+fi
```
Comment (MaxStalker) [inline check-headers.sh:8]: How is this part working? 😅
I'm guessing that we first try to find file with missing line(s) and then if we do - we output them and exit with 1?
And if there are no such filex we exit with 0? 🤔

**Review by MaxStalker — APPROVED** [review]
Looking good 👍

File: check-headers.sh (hunk 1 above)
Comment (turbolent) [reply check-headers.sh:8]: > we first try to find file with missing line(s) and then if we do - we output them and exit with 1?
And if there are no such filex we exit with 0?

Exactly! And CI will abort for exit code != 0

**Pull request merged**